packages/website/src/docs/language-basics/type-literals.md

```diff
@@ -0,0 +1,49 @@
+---
+id: type-literals
+title: Type literals
+---
+
+# Type literals
+
+API authors often need to describe API shapes in terms of specific literal values. For example, this operation returns this specific integer status code, or this model member can be one of a few specific string values. It is also often useful to pass specific literal values to decorators. Cadl supports string, number, and boolean literal values to support these cases.
+
+## String literals
+
+String literals can be represented using double quotes `"`
+
+```cadl
+alias Str = "Hello World!";
+```
+
+## Multi line string literals
+
+A multi string literal is represented using a set of 3 double quotes `"""`.
+
+```cadl
+alias Str = """
+This is a multi line string
+ - opt 1
+ - opt 2
+""";
+```
+
+- Opening `"""` must be followed by a new line.
+- Closing `"""` must be preceded by a new line.
```

Comment:
It would be good to document the indentation handling. Indentation before the ending """ is stripped from each line so that you can indent a multi-line string literal in the Cadl document without indenting its content.

Reply:
Great idea. This made me notice that the formatter isn't being very smart with those. Wondering if it should. https://github.com/microsoft/cadl/issues/1016